src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildWorkspaceUpgradeContext.cs

```diff
@@ -54,7 +57,7 @@ public MSBuildWorkspace Workspace
             }
 
             _projectCache = new Dictionary<string, IProject>(StringComparer.OrdinalIgnoreCase);
-            _path = options.ProjectPath;
+            InputPath = options.ProjectPath;
```

Comment:
This is interesting. I'm guessing this means you automatically receive a `private` setter when you made your property above.

Reply:
Correct.